pkg/frontend/admin_platformworkloadidentityroleset_list.go

```diff
@@ -0,0 +1,49 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"net/http"
+	"path/filepath"
+	"sort"
+
+	"github.com/coreos/go-semver/semver"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/api/admin"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+)
+
+func (f *frontend) getAdminPlatformWorkloadIdentityRoleSets(w http.ResponseWriter, r *http.Request) {
```

Comment:
nit: the file is named `list`, and the only function in it is a `get` (albeit, a plural one ending in `s`). Keeping consistent names helps identify and clearly map code to files and their test cases, so I'd suggest picking one verb for everything. I think you did this perfectly in the `put` files, code, and their tests - can we do that here too?

Reply:
ok, reviewing the rest of our code I am seeing we do this _everywhere_ e.g. `openshiftclusters_list` with a `getOpenshiftClusters` function inside :face_exhaling: Gonna keep the comment for posterity, but nothing to do. Good work following convention.

Reply:
Yep - even though it's a little odd, I did it for the sake of following the convention.